plugins/usbdmx/NodleU1Device.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * NodleU1Device.h
+ * A NodleU1 device that creates an input and an output port.
+ * Copyright (C) 2015 Stefan Krupop
+ */
+
+#ifndef PLUGINS_USBDMX_NODLEU1DEVICE_H_
+#define PLUGINS_USBDMX_NODLEU1DEVICE_H_
+
+#include <memory>
+#include <string>
+#include "ola/base/Macro.h"
+#include "olad/Device.h"
+#include "plugins/usbdmx/NodleU1.h"
+
+namespace ola {
+namespace plugin {
+namespace usbdmx {
+
+/**
+ * @brief A NodleU1 device.
+ *
+ * This device creates a single output port around a Widget.
```

Comment:
Can you correct either this comment or the one above, they appear to conflict currently.